frontend/components/Dialog/ConfigView.tsx

```diff
@@ -204,15 +205,15 @@ function BasicTabs({ onClose }: BasicTabsProps) {
       setSqlite({ ...config.sqlite_options }); // added sqlite
     };
 
-    menuDispatch({
+    dispatch({
```

Comment:
Replace current Context API code with correct redux function calls. You will need to import those functions from the slices